website/our-vision.md

```diff
@@ -1,82 +1,53 @@
-# Our vision
+# Open Documentation Academy
 
-_Discover open source through documentation._
+__Document. Learn. Contribute. Lead: Discover open source through better documentation.__
 
-## Mission
+## Our Vision
 
-Canonical's Open Documentation Academy, or CODA, aspires to be the centre of open source collaboration through documentation.
+__Documentation accessible to all. Contribution possible for everyone. Leadership within reach.__
 
-We create opportunities for people who are new to open source by connecting them with documentation tasks on projects of distinction. We provide assistance and mentoring on these tasks, so that members develop skills in a safe and supportive environment. With our help, your open source journey can be a valuable experience that you can be proud of and receive tangible recognition for.
+At CODA, Canonical Open Documentation Academy, we envision a future where quality documentation is the foundation and lifeblood of open-source ecosystems—a world where clear, accessible, and thoughtful content demystifies technology, inviting both newcomers and experienced contributors from every corner of the globe to collaborate, innovate, and ensure open-source thrives for generations to come.
 
+---
```

Comment:
Can we stick to @degville's version for the heading and introduction?

> # Our ambition for the Open Documentation Academy
> 
> We create opportunities for people who are new to open source by connecting them with documentation tasks on established projects. We provide assistance and mentoring on these tasks, so that members develop skills in a safe and supportive environment.
> 
> With our help, your open source journey can be a valuable experience that you can be proud of and receive tangible recognition for.